### PR TITLE
update json imports in app

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?

**What**: Updates `json` imports. More information about `json` imports in webpack migration docs: https://webpack.js.org/migrate/5/#using-named-exports-from-json-modules

## Screenshots or screen recordings
- N/A just fewer errors in the console.

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #867** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/867" target="_blank">Graphite</a> 
    * **PR #868** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/868" target="_blank">Graphite</a> 
      * **PR #869** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/869" target="_blank">Graphite</a> 
        * **PR #870** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/870" target="_blank">Graphite</a> 
          * **PR #871** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/871" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/871?utm_source=stack-comment).

**Review by johnnyd-eth — APPROVED** [review]
Clean clean clean @benschac LGTM. Will wait for another set of eyes on all these before merging though

**Pull request merged**
